### PR TITLE
Fix typos in comments

### PR DESCRIPTION
**Pull request by jeffhsta**
Fix typo on struct `BTree` documentation and giving a suggestion on `keys` method documentation.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -21,8 +21,8 @@ enum Action {
     RemoveEntry(String),
 }
 
-/// `BTree` is where the informatio `Sender` is contained.
-/// Its inner implementation has a tuple containing the action to be taken as well as a oneshot channel to receive data.
+/// `BTree` is where the information `Sender` is contained.
+/// It's inner implementation has a tuple containing the action to be taken as well as a oneshot channel to receive data.
```
Comment (naomijub) [inline src/lib.rs:25]: Why did you change from its to it's? 

`its inner implementation` would mean something like `sua implementação interna`, while
`it's inner implementation` would mean something like `é a implementação interna`.

Here the correct would be `its`

**Review by naomijub — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (jeffhsta) [reply src/lib.rs:25]: You're right, I missread it.

**Review by naomijub — APPROVED** [review]
